### ISSUE TITLE
Feature request: @Include functionality from RazorEngine

### ISSUE THREAD

**Issue by kae** [toddams/RazorLight#6]
https://antaris.github.io/RazorEngine/LayoutAndPartial.html#Partial-templates

**Comment by kae** [comment]
Sorry to ask, but is there any expectations about beta4 timeframe?

I don't want to push you, just need some clarification so I can select my strategy: wait for new release or start manually including content in current templates.

**Comment by toddams** [comment]
I am on my vacation at the moment and will be home the next week. I think next release will be done by the end of this month (Aug 28-29)

**Comment by toddams** [comment]
The feature is implemented in [1.0.0-beta4](https://www.nuget.org/packages/RazorLight/1.0.0-beta4)

Example
````CSharp
@model MyProject.TestViewModel
<div>
    Hello @Model.Title
</div>

@{ await IncludeAsync("SomeView.cshtml", Model); }
````
First argument takes a key of the template to resolve, second argument is a model of the view to include (can be null)

_Note: there is only an async version of this method_
